src/components/sync-sites-modal-selector.tsx

```diff
@@ -38,7 +38,7 @@ export function SyncSitesModalSelector( {
 			title={ __( 'Connect a WordPress.com site' ) }
 		>
 			<SearchSites searchQuery={ searchQuery } setSearchQuery={ setSearchQuery } />
-			<div className="h-[calc(84vh-230px)]">
+			<div className="h-[calc(84vh-232px)]">
```

Comment:
Slightly annoying that the `Modal` from `@wordpress/components` makes it so difficult to set the height of this element dynamically. This seems like a pragmatic solution given those constraints, though.